bin/filter-preview-output.sh

```diff
@@ -6,4 +6,6 @@ grep -v -e '^\.\.\.0 corrections$' |\
   grep -v -e '\.\.\. (skipping)' |\
   grep -v -e '^----- DNS Provider: ' |\
   grep -v -e '^----- Registrar: ' |\
-  grep -v -e '^----- Getting nameservers from:'
+  grep -v -e '^----- Getting nameservers from:' | \
+  sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2};?)?)?[mGK]//g" # remove ANSI color codes
```

Comment:
Instead of the sed, you can use ``--no-color` (it goes before the "preview" or "push" verb).

I would also add:

* `--reportmax=0` -- be less verbose

Reply:
The reason I implemented it like this was so that colors still worked in the action log, but weren't commented in the PR.

Reply:
ah!  makes sense.

(it's a shame that whoever added the `-p` flag to `col` didn't also strip escape sequences)